modin/core/dataframe/pandas/dataframe/utils.py

```diff
@@ -122,6 +124,10 @@ class ShuffleSortFunctions(ShuffleFunctions):
         The ideal number of new partitions.
     level : list of strings or ints, or None
         Index level(s) to use as a key. Can't be specified along with `columns`.
+    closed_on_right : bool, default: False
```

Comment:
```suggestion
    close_to_right : bool, default: False
```
?

Reply:
here I refer to the term ["closed interval"](https://mathworld.wolfram.com/ClosedInterval.html), `closed_on_right` means, that we have to include the right bound in it